src/sidebar/services/annotations-exporter.tsx

```diff
@@ -152,6 +153,90 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    return renderToString(
+      <html lang="en">
+        <head>
+          <title>Annotations export - Hypothesis</title>
+          <meta charSet="UTF-8" />
+        </head>
+        <body>
+          <header>
+            <p>
+              <time dateTime={now.toISOString()}>{formatDateTime(now)}</time>
+            </p>
+            <p>
+              <strong>{title}</strong>
+            </p>
+            <p>
+              <a href={uri} target="_blank" rel="noopener noreferrer">
+                {uri}
+              </a>
+            </p>
+
+            <dl>
+              <dt>Group</dt>
+              <dd>{groupName}</dd>
+              <dt>Total users</dt>
+              <dd>{uniqueUsers.length}</dd>
+              <dt>Users</dt>
+              <dd>{uniqueUsers.join(', ')}</dd>
+              <dt>Total annotations</dt>
+              <dd>{annotations.length}</dd>
+              <dt>Total replies</dt>
+              <dd>{replies.length}</dd>
+            </dl>
+          </header>
+          <hr />
+          <section>
+            {annotations.map((annotation, index) => {
+              const page = pageLabel(annotation);
+              return (
+                <Fragment key={annotation.id}>
+                  <article>
+                    <strong>Annotation {index + 1}:</strong>
+                    <p>
+                      <time dateTime={annotation.created}>
+                        {formatDateTime(new Date(annotation.created))}
+                      </time>
+                    </p>
+                    <p>Comment: {annotation.text}</p>
+                    <p>{extractUsername(annotation)}</p>
+                    <p>
+                      Quote: <blockquote>{quote(annotation)}</blockquote>
+                    </p>
+                    {annotation.tags.length > 0 && (
+                      <p>Tags: {annotation.tags.join(', ')}</p>
+                    )}
+                    {page && <p>Page: {page}</p>}
+                  </article>
+                  <br />
```

Comment:
What is the `<br />` for? `<article>` elements are blocks so there should be a gap between them automatically.

Reply:
Just to add some extra space between every annotation block, otherwise there's the same space between every line in one annotation, and the first line on the next one.

On the other hand, probably after applying [this suggestion](https://github.com/hypothesis/client/pull/6073#discussion_r1448721059), this becomes less problematic and we can get rid of the `<br />`s